Redirect/PluginUI.cs

```diff
@@ -176,11 +176,11 @@ class PluginUI : IDisposable {
             ImGui.End();
         }
 
-        private TextureWrap? FetchTexture(ushort id) {
-            Icons.TryGetValue(id, out TextureWrap? texture);
+        private IDalamudTextureWrap? FetchTexture(ushort id) {
+            Icons.TryGetValue(id, out IDalamudTextureWrap? texture);
 
-            if (texture is null && id > 0) {
-                texture = Plugin.DataManager.GetImGuiTextureIcon(id);
+            if (texture is null && id > 0) {;
```

Comment:
```suggestion
            if (texture is null && id > 0) {
```
Is this semicolon supposed to be here?